### PR TITLE
Revisions issues: RDoc-2182, RDoc-2186, RDoc-2181

### PR DESCRIPTION
**Pull request by reebhub**
First commit: RDoc-2182 - updated conflict revisions snapshots

### CODE REVIEW

File: Documentation/5.3/Raven.Documentation.Pages/document-extensions/revisions/overview.markdown [hunk 1]
```diff
@@ -313,6 +313,17 @@ public enum ForceRevisionStrategy
...
     {CODE-TABS/}
 
+
+{NOTE: When the Revisions feature is disabled:}
+
+Deleting a document that owns a manually-created revision 
+will not move the revision to the revisions bin.  
+* a `delete revision` will Not be created.  
+* manual revisions will not be kept in the revisions bin.  
+
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/document-extensions/revisions/overview.markdown:323]: 1. Missing spaces before the 'stars' ?
The UI doesn't show the 'regular bullets'

2. Can you pls capitalize each bullet? 
a =>A 
manual=> Manual

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/document-extensions/revisions.markdown [hunk 2]
```diff
@@ -106,11 +106,16 @@ revisions to the live document version.
 
 {PANEL: Revisions Bin}
 
-Deleting a document that the Revisions feature is enabled for will:  
+When the Revisions feature is **Enabled** for a document's collection, and the document is deleted:  
+
+* A delete-revision will be created for the document, marking its deletion.  
+* The delete-revision and all the document's revisions, both Automatically-Created 
+  and Manually-Created, will be moved to the Revisions Bin.
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/studio/database/document-extensions/revisions.markdown:113]: Need to explain:

If the deleted document has ONLY manually-created revisions then they are Not moved to the Revisions Bin.

Revisions are moved if the document has either:
**Auto-created revisions only**
or: 
**Auto-created+Manual**

..If only manual then they are not moved even if feature is enabled..

**Review by Danielle9897 — CHANGES_REQUESTED** [review]

File: Documentation/5.3/Raven.Documentation.Pages/document-extensions/revisions/overview.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/document-extensions/revisions/overview.markdown:323]: done

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/document-extensions/revisions.markdown (hunk 2 above)
Comment (Danielle9897) [reply Documentation/5.3/Raven.Documentation.Pages/studio/database/document-extensions/revisions.markdown:113]: Run the following:

1. don't set any no revisions configuration
2. create a few manual revisions on a document
3. delete the document
4. the document+the manual revisions do Not go to the revisions bin

**Review by Danielle9897 — APPROVED** [review]

**Pull request merged**